deptofdefense/move.mil
# Update gems

## Checklist

I have…


## Summary of Changes

This pull request updates several development- and test-specific gems (brakeman, rspec-rails, and simplecov) to their latest versions.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. `git checkout update-gems`,
1. set up development dependencies according to `CONTRIBUTING.md`,
1. `bin/brakeman`,
1. `bin/rake`,
1. `bin/rails server`, and
1. load up http://localhost:3000 in your Web browser of choice.
